source/reference/failure-recovery.txt

```diff
@@ -0,0 +1,96 @@
+.. _c2c-failure-recovery:
+
+================
+Failure Recovery 
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+:program:`mongosync` temporarily alters some collection characteristics
+during synchronization. The original values are restored during the
+:ref:`commit process <c2c-api-commit>`. However, a network failure or
+some other problem could disrupt synchronization before the sync is
+committed. If the sync ends prematurely, the temporary changes must be
+manually updated on the destination cluster.
+
+Manually revert the following temporary changes:
```

Comment:
[suggestion] Because the fragments in this unordered list don't result in sentences when added to the introductory phrase, I don't think the four items here need periods.

Reply:
Done